Bottom: 8d34dc7478e2f935378c6aa843c3807bd332b540
Top:    29a597da3fdf57d2c60d22b25dab52fdd8113edc
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-02 17:55:49 +0800

vhost: bypass wrap_counter settings

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 26e21af..f3697b5 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1426,10 +1426,12 @@ long vhost_vring_ioctl(struct vhost_dev *d, unsigned int ioctl, void __user *arg
 		vq->last_avail_idx = s.num;
 		/* Forget the cached index value. */
 		vq->avail_idx = vq->last_avail_idx;
+#if 0
 		if (vhost_has_feature(vq, VIRTIO_F_RING_PACKED)) {
 			vq->last_avail_wrap_counter = wrap_counter;
 			vq->avail_wrap_counter = vq->last_avail_wrap_counter;
 		}
+#endif
 		break;
 	case VHOST_GET_VRING_BASE:
 		s.index = idx;
@@ -1460,8 +1462,10 @@ long vhost_vring_ioctl(struct vhost_dev *d, unsigned int ioctl, void __user *arg
 			break;
 		}
 		vq->last_used_idx = s.num;
+#if 0
 		if (vhost_has_feature(vq, VIRTIO_F_RING_PACKED))
 			vq->last_used_wrap_counter = wrap_counter;
+#endif
 		break;
 	case VHOST_GET_VRING_USED_BASE:
 		s.index = idx;
